vmware_exporter/vmware_exporter.py

```diff
@@ -489,47 +498,50 @@ def _vmware_get_datastores(self, ds_metrics):
         Get Datastore information
         """
 
-        log("Starting datastore metrics collection")
         results, datastore_labels = yield parallelize(self.datastore_inventory, self.datastore_labels)
 
         for datastore_id, datastore in results.items():
-            name = datastore['name']
-            labels = datastore_labels[name]
-
-            ds_capacity = float(datastore.get('summary.capacity', 0))
-            ds_freespace = float(datastore.get('summary.freeSpace', 0))
-            ds_uncommitted = float(datastore.get('summary.uncommitted', 0))
-            ds_provisioned = ds_capacity - ds_freespace + ds_uncommitted
+            try:
```

Comment:
Is this try just do catch where `datastore_labels` is missing a key or where `datastore` is missing a key? To avoid the nesting, would it be enough to do:

```python
try:
    name = datastore['name']
    labels = datastore_labels[name]
except KeyError:
    logging.exception(f'Datastore label for {name} missing')
    continue
```

Reply:
Implemented in the most up to date version